example/tutorial/train_diabetes.py

```diff
@@ -0,0 +1,159 @@
+#
+# train_diabetes.py
+#
+#   Modified version of train.py from Wine Quality dataset
+#       https://github.com/databricks/mlflow/blob/master/example/tutorial/train.py
+#       The data set used in this example is from http://archive.ics.uci.edu/ml/datasets/Wine+Quality
+#       P. Cortez, A. Cerdeira, F. Almeida, T. Matos and J. Reis.
+#       Modeling wine preferences by data mining from physicochemical properties. In Decision Support Systems, Elsevier, 47(4):547-553, 2009.
+#   Combines the above with the Lasso Descent Path Plot
+#       http://scikit-learn.org/stable/auto_examples/linear_model/plot_lasso_coordinate_descent_path.html
+#       Original author: Alexandre Gramfort <alexandre.gramfort@inria.fr>; License: BSD 3 clause
```

Comment:
Can you also update the comment to say what this is instead of saying it's based on the Wine one? It doesn't use the wine data, right? Just describe it from first principles. We didn't get any code from an existing site for wine, we just got the data.